tso-server/src/main/java/com/yahoo/omid/tso/MonitoringContext.java

```diff
@@ -29,7 +28,7 @@
 
 import static com.yahoo.omid.metrics.MetricsUtils.name;
 
-@NotThreadSafe
+//NotThreadSafe
```

Comment:
When updating Guava from 11.x to 14.x those annotations dissapeared, so I substituted the marker annotation by a comment. That was the only purpose of that